client/webserver/site/src/html/forms.tmpl

```diff
@@ -185,6 +193,20 @@
 <div class="fs15 pt-3 text-center d-hide errcolor text-break" data-tmpl="err"></div>
 {{end}}
 
+{{define "discoverAcctForm"}}
+<div class="py-1 text-center fs28 sans-light offwhite lh1" data-tmpl="header">[[[Create Account]]]</div>
+<div class="fs16 text-center mb-4">on <span data-tmpl="dexHost"></span></div>
+<div data-tmpl="appPWBox" class="pb-2">
+  <label for="discoverAcctAppPW" class="form-label ps-1 mb-1">[[[App Password]]]</label>
+  <div class="fs14 px-1 mb-1">Your app password is always required when performing sensitive operations.</div>
```

Comment:
If we keep this message, it needs a locale template entry.  However, I don't think it's needed.  Regarding the message, other operations only show the password text box if the pass is not cached, so why is this form more sensitive than e.g. registration?